config.h.meson

```diff
@@ -0,0 +1,98 @@
+/* config.h.in.  Generated from configure.ac by autoheader.  */
+
+/* Define to 1 if you have the <inttypes.h> header file. */
+#mesondefine HAVE_INTTYPES_H
+
+/* Define to 1 if you have the <memory.h> header file. */
+#mesondefine HAVE_MEMORY_H
+
+/* Define if SELinux is available */
+#mesondefine HAVE_SELINUX
```

Comment:
That is what it's for. If you do `cdata.set('HAVE_SELINUX', 1)` then that will get replaced by

    #define HAVE_SELINUX 1

If it is not set, then it will be replaced with

    /* undef HAVE_SELINUX */

As mentioned above I don't have an SELinux machine to test this with so I left it out.